src/main/java/marquez/core/exceptions/UnexpectedErrorException.java

```diff
@@ -0,0 +1,3 @@
+package marquez.core.exceptions;
+
+public class UnexpectedErrorException extends Throwable {}
```

Comment:
Is `UnexpectedErrorException` viewed more as a placeholder exception until more of the service layer is defined?

Reply:
I think this is the approach we agreed upon.

Reply:
@wslulciuc we will have the stacktrace to pin point where the error was in the stack. `UnexpectedErrorException` can be a really basic exception that all services will throw (in the absence of more specific exceptions) for now so that Aleks can move forward.

Reply:
Also, can we rename this to `UnexpectedException`? `UnexpectedErrorException` is a little confusing since it errors and exceptions are not exactly the same thing.